### DISCUSSION TITLE
What about setting Obsidian's save location to one of OneDrive's folders instead of using this extension?

### DISCUSSION THREAD

**Discussion by Luomint** [remotely-save/remotely-save#375] [General]
What about setting Obsidian's save location to one of OneDrive's folders instead of using this extension? Is there a reason to use this extension?

Add that it's possible even with the free version of OneDrive, and to set Obsidian's folder to be always available, even offline, so it doesn't disappear during periods of non-use with the free version.

The creator of this extension also incorrectly implies that none of the cloud services are free when I've been under the limit by using Google Drive and OneDrive for a decade.

**Comment by berot3** [comment]
> The creator of this extension also incorrectly implies that none of the cloud services are free

Can you quote where he implies that? Couldn't find it now.

**Comment by berot3** [comment]
It would be great to have some kind of table for a quick comparison. But the main reasons I think are that you can also self-host and you don't give the data to some big companies. Except you encrypt them before storing in the cloud. 

Also the sync seems almost instant, but I still have to test this myself 😄

Also: https://github.com/remotely-save/remotely-save/issues/280#issuecomment-1528642132

**Comment by fyears** [comment]
one major concern is that i want a almost free and stable sync on ios which is hard to combine obsidian and other apps
